profiling/src/exporter/mod.rs

```diff
@@ -150,6 +151,22 @@ impl ProfileExporter {
         })
     }
 
+    #[cfg(target_env = "musl")]
+    fn runtime_platform_tag(&self) -> Tag {
+        tag!(
+            "runtime_platform",
+            concatcp!(std::env::consts::ARCH, "-", std::env::consts::OS, "-musl")
+        )
+    }
+
+    #[cfg(not(target_env = "musl"))]
+    fn runtime_platform_tag(&self) -> Tag {
+        tag!(
+            "runtime_platform",
+            concatcp!(std::env::consts::ARCH, "-", std::env::consts::OS)
+        )
+    }
+
```

Comment:
Nit: just as suggestion. Probably personal preference have you thought about to define the runtime `str` per target as in:
```
#[cfg(target_env = "musl")]
const RUNTIME_INFO: &str = concatcp!(std::env::consts::ARCH, "-", std::env::consts::OS, "-musl");
#[cfg(not(target_env = "musl"))]
const RUNTIME_INFO: &str = concatcp!(std::env::consts::ARCH, "-", std::env::consts::OS);
```
?

and have just one function defined:
```
    fn runtime_platform_tag(&self) -> Tag {
        tag!( "runtime_platform", RUNTIME_INFO)
    }

```
I find this approach less cluttering and if you end up with many specific definitions per target is easy to move them to separate files and have them included by cfg attributes